### PR TITLE
Fix Backbone Cable visual bugs

### PR DESCRIPTION
**Pull request by Pilzinsel64**
While searching throw the code I found a few little bugs(?) and suggest a fix via this PR. This are the only few I found, anything else seems stable on a first review and while testing them.

I'm not sure if there is wrong anything else, but I heard from a few people that they are disabled due some bugs. Any idea what kind of bugs that are? Maybe we can fix them in this PR too (or in another one). Just let me know! _(Feel free to ping me on Discord.)_

The only thing I would give to discuss is that point here at _"Maybe a bug"_: https://github.com/GTNewHorizons/EnderIO/pull/105#issuecomment-1454674199

### CODE REVIEW

**Review by repo-alt — APPROVED** [review]

**Comment by repo-alt** [comment]
Unfortunately bugs in dense cables support are really hard to reproduce. They are usually reproduced when you have some complicated topology with hundreds of nodes. But, because every engame base has hundreds of nodes, we can't really enable dense cables in config by defaut.
The main reason is that initial path finding algorithm was not design to deal with the cables able to pass channels to several controller sides, so I had to hack quite a bit to circumvent that. The easiest solution would be of course just buff controllers to have 128 channels at every side but that looks like a boring powercreep.

**Comment by Auge103** [comment]
> I'm not sure if there is wrong anything else, but I heard from a few people that they are disabled due some bugs. Any idea what kind of bugs that are?

Not a GTHN dev, but let me explain what happened when I replaced my main dense cable with backbone ones.
[I was using 2.3.0.0-RC3];

After I replaced the cables, the backbone ones immediately showed the correct amount of channels and connected everything that was attached to them with the controller. This was about ~30 ME P2P bus devices.
The network came back online.
The strange part:
The devices behind the ME P2P devices all came back online as well (showed "online" and blue), but all crafting recipes from the interfaces weren't available in the terminal.
Even more strange, all the dense cables behind the P2P devices did show "0 channels" and no blue lines, despite having all 32 channels maxed on them and each and every device online.

All import/export bus did do their job just fine, but the cables (and the network visualisation tool) still did show zero channels online.

So to summarize it all, the devices behind P2P were online, did have a channel, did NOT show crafting recipes and the cables they were connected to did show 0 channels.
While the main line worked fine.

I restarted my server and all the recipes did show up again, but the dense cables still were at 0 channels.
I then reverted everything and replaced the backbone cables with dense ones like before and everything started working as it should.
So clearly the backbone cables are messing with P2P devices somehow.

Should I write a bug report about this?

**Comment by Pilzinsel64** [comment]
@Auge103 Maybe set this config value to true? Not sure if that helps.
grafik
